jscatter/encodings.py

```diff
@@ -5,14 +5,44 @@
 from math import floor
 from typing import List, Tuple, Union
 
-def create_legend(values, encoding, norm, categories, linspace_num=5):
+def create_legend(encoding, norm, categories, labeling=None, linspace_num=5):
+    variable = None
+    if labeling:
+        try:
+            variable = labeling['variable']
+        except KeyError as e:
+            variable = None
```

Comment:
I think this logic can be simplified with the builtin `.get` method. Rather than raising an error, a fallback value is returned if the key is missing (`None` by default).

```suggestion
    variable = labeling.get('variable') if labeling else None
```

Reply:
So true! Good catch